recipes-extended/qemu-dm/qemu-dm.inc

```diff
@@ -72,6 +72,8 @@ S = "${WORKDIR}/qemu-${PV}"
 
 inherit autotools-brokensep pkgconfig python3native
 
+require recipes-extended/qemu-dm/qemu-dm-vglass.inc
```

Comment:
Should meta-vglass just bbappend qemu-dm?  Broader question, do we want all of https://github.com/OpenXT/xenclient-oe/pull/1382/commits/c3e733acd4c72403850236812ba5325020121255 to be bbappends in meta-vglass?

Reply:
The reason why I ended up doing this is to keep `xenclient-oe` depending on `meta-vglass`, not forcing `meta-vglass` to use `xenclient-oe`.
The `qemu-dm` recipes are `xenclient-oe` specific, but the patches might be of use to someone who wants to use a different QEMU device-model recipe, and are vglass specific, so an include seemed like the best compromise.

Reply:
Okay, makes sense.